src/v2/Apps/Artist/Routes/Articles/ArtistArticles.tsx

```diff
@@ -111,6 +111,7 @@ export class ArtistArticles extends Component<
           <Col>
             <Box>
               <Pagination
+                getHref={() => ""}
```

Comment:
Any reason why this isn't the default prop?

Reply:
I think it makes way more sense to pass the page query params rather than empty string. IMO everything that's using this is doing it wrong since it breaks URL nav/back button etc.

Reply:
Yeah, ideally everything should be URL driven, which then cascades down to various bits of state.